### PR TITLE
Domain classes + Mapper class

### PR DESCRIPTION
**Pull request by zajacamadeusz**
Game
GameDto
GameMapper

### CODE REVIEW

File: src/main/java/com/kodigames/domain/Game.java [hunk 1]
```diff
@@ -15,8 +15,8 @@
     public Game() {
     }
 
-    public Game(Long gameId, String name, String desc) {
-        this.gameId = gameId;
+    public Game(long gameId, String name, String desc) {
+        this.id = gameId;
```
Comment (paulinakuzmicka) [inline src/main/java/com/kodigames/domain/Game.java]: gameId -> id

File: src/main/java/com/kodigames/domain/Game.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 @Entity
 @Table(name = "GAMES")
 public class Game {
-    private Long gameId;
```
Comment (maciolek) [inline src/main/java/com/kodigames/domain/Game.java]: final do class

File: src/main/java/com/kodigames/domain/Game.java [hunk 3]
```diff
@@ -25,8 +25,8 @@ public Game(Long gameId, String name, String desc) {
     @GeneratedValue
     @NotNull
     @Column(name = "ID")
```
Comment (maciolek) [inline src/main/java/com/kodigames/domain/Game.java]: uniqe=true

File: src/main/java/com/kodigames/domain/Game.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    @NotNull
+    @Column(name = "NAME")
+    public String getName() {
+        return name;
+    }
+
+    @NotNull
+    @Column(name = "DESCRIPTION")
+    public String getDesc() {
+        return desc;
```
Comment (MalczewskiKarol) [inline src/main/java/com/kodigames/domain/Game.java:39]: add setters to id, name, desc

**Review by maciolek — DISMISSED** [review]

**Review by paulinakuzmicka — CHANGES_REQUESTED** [review]

**Review by MalczewskiKarol — CHANGES_REQUESTED** [review]
bez setterow nie pojedzie :)

**Review by paulinakuzmicka — DISMISSED** [review]
Ok :)

**Review by MalczewskiKarol — APPROVED** [review]

**Pull request merged**